ReactAndroid/src/main/java/com/facebook/react/modules/dialog/AlertFragment.java

```diff
@@ -84,4 +75,8 @@ public void onDismiss(DialogInterface dialog) {
       mListener.onDismiss(dialog);
     }
   }
+
+  /*package*/ void setAlertFragmentListener(@Nullable DialogModule.AlertFragmentListener alertFragmentListener) {
```

Comment:
i haven't worked with java that much. is `/*package*/` required? @hramos @gengjiawen